packages/lumos/src/config.ts

```diff
@@ -16,4 +16,13 @@ export {
   validateConfig,
   initializeConfig,
   getConfig,
+  /**
+   * @deprecated use the {@link nameOfScript} and {@link findConfigByScript} function instead
+   */
+  helpers,
```

Comment:
Why not add the mark where `helpers` is defined

Reply:
The submodule named `helpers` is defined in `@ckb-lumos/config-manager/lib/helpers`, and there was a `helpers` already in `@ckb-lumos/helpers`, this may confuse.

```ts
import { helpers } from '@ckb-lumos/lumos' // OK

import { helpers } from '@ckb-lumos/lumos/config' // Not recommended
import { nameOfScript } from '@ckb-lumos/lumos/config' // Recommended

```

Reply:
I mean why not mark `deprecated` above `helpers` in `@ckb-lumos/lumos/config`

Reply:
you're right, the original helpers should also be marked as deprecated for more clear

Reply:
@Keith-CY Can you mark this comment as resolved if you think it's OK?